build/ical.js

```diff
@@ -886,7 +894,9 @@ ICAL.design = (function() {
 
   // When adding a value here, be sure to add it to the parameter types!
   var vcardValues = ICAL.helpers.extend(commonValues, {
-
+    component: textType(FROM_ICAL_NEWLINE, TO_ICAL_NEWLINE, "text"),
```

Comment:
Sorry I missed this when I took an initial look, but can you tell me more about this new `component` type? I assume this is not a real type that will show up in the jCal, but instead `text` will show. Why do structured text values have different newline properties than normal text properties?